openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,38 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            resources = {}
+            extra_values = {}
+            if 'load' in data['resources']:
+                extra_values['load'] = {
+                    'load_1': 100 * data['resources']['load'][0],
+                    'load_15': 100 * data['resources']['load'][2],
+                }
+                resources['load'] = 100 * data['resources']['load'][1]
```

Comment:
@PabloCastellano reminded me that we need to know the number of processors in order to calculate the CPU usage, read here: https://blog.appsignal.com/2018/03/28/understanding-system-load-and-load-averages.html

Can you add this information to the payload that it's sent?

On OpenWRT you can get the number of processors with:

```
cat /proc/cpuinfo | grep -c processor
```

(@PabloCastellano `numprocs` is not available by default on OpenWRT)